server/api/role/list/service.js

```diff
@@ -68,6 +83,10 @@ async function listRoles(db, { q, limit, cursor, scopes }) {
   return roles.map((role) => ({
     id: role._id.toHexString(),
     name: role.name,
+    org: {
+      id: get(role, 'org._id') && get(role, 'org._id').toHexString(),
```

Comment:
I suggest to change this as follows...

```
org: has(role, ['org', '_id']) ? {
  id: role.org._id.toHexString(),
  name: role.org.name,
} : null
```

Here's my reasoning:

1. If `org._id` doesn't exist then the `org` property should be null (as we discussed on slack);
2. You needn't check for `org.name` and `org._id` separately. If `org._id` exists then there is definitely an `org.name` as well.